src/structs/BPTokenMapping.sol

```diff
@@ -1,7 +1,7 @@
-// SPDX-License-Identifier: UNLICENSED
+// SPDX-License-Identifier: MIT
 pragma solidity ^0.8.21;
 
-struct BPTokenConfig {
+struct BPTokenMapping {
```

Comment:
@filipviz why "mapping"? dont we use "config" in core?

Reply:
Because in context, this struct is used to map a token on the local chain to the remote chain. I use the verb "map" throughout. Maybe this is confusing – we can replace with "bind" "link" but I like the directionality of mapping *to* something

Reply:
"token config" is vague and doesn't really convey meaning imo